app/javascript/stylesheets/typography.css

```diff
@@ -10,4 +10,8 @@
   p {
     @apply text-gray-500 text-lg tracking-wide;
   }
+
+  .toc-item-active {
```

Comment:
Add to lesson-content.css after https://github.com/TheOdinProject/theodinproject/pull/3137 is merged